modules/core/src/main/java/org/apache/ignite/internal/processors/cache/persistence/pagemem/PagesWriteSpeedBasedThrottle.java

```diff
@@ -225,6 +226,11 @@ public PagesWriteSpeedBasedThrottle(
             doPark(throttleParkTimeNs);
         }
 
+        DataRegionMetricsImpl metrics = pageMemory.metrics();
+
+        if (metrics != null)
```

Comment:
Can the metric instance be `null`?

Reply:
I have implemented NO-OP metrics for some tests. Now it can't be null.